mapillary/utils/format.py

```diff
@@ -376,3 +379,205 @@ def geojson_to_polgyon(geojson: dict) -> dict:
+
+    :param coordinates: The coordinate list to normalize
+    :type coordinates: list
+
+    :param width: The width of the coordinates to normalize with, defaults to 4096
+    :type width: int
+
+    :param height: The height of the coordinates to normalize with, defaults to 4096
+    :type height: int
+
+    :return: The normalized list
+    :rtype: list
+    """
+
+    # Extracting the list from the coordinates
+    coordinates = coordinates[0]
+
+    # Initializing the coordinate list
+    new_coordinates = []
+
+    # Going through each coordinate pair
+    for coordinate_pair in coordinates:
+
+        # If it is already normalized ...
+        if 0 <= coordinate_pair[0] <= 1 and 0 <= coordinate_pair[1] <= 1:
+
+            # ... then append as is
+            new_coordinates.append(coordinate_pair)
+
+        # Appending the coordinates
+        new_coordinates.append(
+            # Appending a list pair of the width, height
+            [coordinate_pair[0] / width, coordinate_pair[1] / height]
+        )
+
+    # Returning the results
+    return [new_coordinates]
+
+
+def decode_pixel_geometry(
+    base64_string: str, normalized: bool = False, width: int = 4096, height: int = 4096
+) -> dict:
+    """Decodes the pixel geometry, and return the coordinates, which can be specified to be
+    normalized
+
+    :param base64_string: The pixel geometry encoded as a vector tile
+    :type base64_string: str
+
+    :param normalized: If normalization is required, defaults to False
+    :type normalized: bool
+
+    :param width: The width of the pixel geometry, defaults to 4096
+    :type width: int
+
+    :param height: The height of the pixel geometry, defaults to 4096
+    :type height: int
+
+    :return: A dictionary with coordinates as key, and value as the normalized list
+    :rtype: list
+    """
+
+    # The data retrieved after being decoded as base64
+    data = base64.decodebytes(base64_string.encode("utf-8"))
+
+    # Getting the results from mapbox_vector_tile
+    result = mapbox_vector_tile.decode(data)
+
+    # Flattening the resultant dicionary
+    flattened = flatten_dictionary(result)
+
+    # Init geometry var
+    geometry = None
+
+    # For key in the flattend keys
+    for key in flattened.keys():
+
+        # If the key contains the word "features"
+        if "features" in key:
+
+            # Get the geometry
+            geometry = flattened[key][0]
+
+    # Extract the coordinate list
+    coordinate_list = geometry["geometry"]["coordinates"]
+
+    # Return output
+    return (
+        # Return coordinates as normalized values, if normalized is true
+        {"coordinates": normalize_list(coordinate_list, width=width, height=height)}
+        if normalized
+        # Else return un-normalized coordinates
+        else {"coordinates": coordinate_list}
+    )
+
+
+def decode_pixel_geometry_in_geojson(
+    geojson: typing.Union[dict, GeoJSON],
+    normalized: bool = False,
```

Comment:
Does this mean it will not normalize by default?

Reply:
No, by default it does not normalize. Should I change this to 'True' instead?